.env.production

```
@@ -0,0 +1,65 @@
+# ClaudeOps Production Environment Configuration
+# This file contains production-specific environment variables
+
+# Application Configuration
+NEXT_PUBLIC_APP_URL=https://claudeops.yourdomain.com
+NEXT_PUBLIC_APP_NAME=ClaudeOps
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_APP_NAME key should go before the NEXT_PUBLIC_APP_URL key

(UnorderedKey)